microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -178,114 +251,284 @@ The color of the `<h1>` element is still set to the new color.  The change remai
 
-1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS) to save the change.
+1. In the the `<h1>` tag, replace "DevTools Workspaces Demo" with "I Love Cake".
+
+   An asterisk appears next to `index.html`.
+
+1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS).
+
+   Your change is saved, and the asterisk disappears.
+
+   You made the change in the file listing, rather than in the DOM tree in the **Elements** tool.
 
 1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `app` directory:
 
-   ![Change HTML from the Sources tool](./index-images/workspaces-workspaces-demo-sources-page-h1.png)
+   ![Change HTML from the Sources tool](./index-images/sources-page-h1.png)
+
+1. In the **Sources** tool, right-click `index.html`, and then select **Open in containing folder**.
+
+   File Explorer or Finder opens.
 
 1. Open the copy of `index.html` that's in your `app` directory in a text editor, such as Visual Studio Code.
 
-   The `<h1>` element contains the new text, because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Filesystem** tab), indicated by a green dot on the file's icon.
+   The `<h1>` element contains the new text, "I Love Cake", because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Workspace** tab), indicated by a green dot on the file's icon.
+
+Continue with the next section.
 
 
 <!-- ====================================================================== -->
-## Edit JavaScript and save changes to the source file
+## Step 4: Save a JavaScript change to disk
+<!-- https://developer.chrome.com/docs/devtools/workspaces/#js -->
+<!--
+subsection order for Edge doc:
+1. Open the Quick source tool without using Command Menu.
+2. Open the Quick source tool by using Command Menu.
+3. Open a .js file in the Quick source tool without using Command Menu.
+4. Open a .js file in the Quick source tool by using Command Menu.
+-->
+
+Next, edit JavaScript and save changes to the source file.
 
 The main place to use the code editor of DevTools is the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console**, while editing files.  The **Quick source** tool gives you just the editor from the **Sources** tool, while any tool is open.
-<!-- todo: avoid Quick source tool and just use Sources tool for this section -->
 
-To open the DevTools code editor alongside other tools:
+There are a couple ways to open the **Quick source** tool, and a couple ways to open a file in the **Quick source** tool.  We'll walk through all the ways.
+
+
+<!-- ------------------------------ -->
+#### Open the Quick source tool without using Command Menu
```

Comment:
maybe a better, tighter sequence than
![image](https://github.com/user-attachments/assets/611d2424-4f8c-4653-a374-7b222645a681)
would be two h4s:

Open Quick Source tool & a .js file [w/o using Command Menu]
Open Quick Source tool & a .js file by using Command Menu

@captainbrosset

Reply:
done.  it's better, and matches closely w/ upstream outline